taxi/views.py

```diff
@@ -85,6 +130,24 @@ class DriverListView(LoginRequiredMixin, generic.ListView):
     model = Driver
     paginate_by = 5
 
+    def get_context_data(self, *, object_list=None, **kwargs) -> dict:
+        context = super().get_context_data(**kwargs)
+        context["search_form"] = DriverSearchForm(initial={
+            "username": self.request.GET.get("username", "")
+        })
+        return context
+
+    def get_queryset(self) -> QuerySet:
+        queryset = get_user_model().objects.all()
```

Comment:
The queryset in `DriverListView` currently includes all users. If your user model includes non-driver users, this might not be appropriate. Consider filtering the queryset to include only users who are drivers. You might need to use a specific field or method to filter only driver users.